domain/schema/controller.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/juju/juju/domain/schema/controller/triggers"
 )
 
-//go:generate go run ./../../generate/triggergen -db=controller -destination=./controller/triggers/cloud-triggers.gen.go -package=triggers -tables=cloud,cloud_credential,external_controller
+//go:generate go run ./../../generate/triggergen -db=controller -destination=./controller/triggers/cloud-triggers.gen.go -package=triggers -tables=cloud,cloud_credential,external_controller,cloud_credential_attribute,cloud_ca_cert
```

Comment:
My OCD would like to ask can you group these near the other cloud names in order?

Reply:
I moved the external controllers one to the controller triggers line as well.